apis/src/pages/profile_view.rs

```diff
@@ -21,24 +31,74 @@ pub fn ProfileView() -> impl IntoView {
                 .unwrap_or_default()
         })
     };
-
     let user = Resource::new(username, move |_| get_user_by_username(username()));
+    let games = Resource::new(username, move |_| get_user_games(username()));
+    let stored_children = store_value(children);
+    let button_style = "hover:bg-pillbug-teal transform transition-transform duration-300 active:scale-95 text-white font-bold py-2 px-4 m-1 rounded";
+    let tab_view = create_rw_signal(ProfileGamesView::Playing);
+    let is_playinng = move || {
+        format!(
+            "{button_style} {}",
+            if matches!(tab_view(), ProfileGamesView::Playing) {
+                "bg-pillbug-teal"
+            } else {
+                "bg-ant-blue"
+            },
+        )
+    };
+    let is_finished = move || {
+        format!(
+            "{button_style} {}",
+            if matches!(tab_view(), ProfileGamesView::Finished) {
+                "bg-pillbug-teal"
+            } else {
+                "bg-ant-blue"
+            },
+        )
+    };
+    provide_context(tab_view);
 
     view! {
         <div class="bg-light dark:bg-dark pt-12">
             <Transition>
                 {move || {
+                    let maybe_games = games().map(|games| games.ok()).flatten();
+                    let games = match maybe_games {
```

Comment:
I don't think this is required, you get can just do the partition and do

```
let games = {
  maybe_games.sort_by... 
  maybe_games.partition...
}
```
that should yield the same as you match here